scripts/ft-benchmark-data-sender.py

```diff
@@ -85,45 +87,25 @@ def get_commit() -> tuple[str, datetime]:
     return (full_commit_hash, commit_timestamp)
 
 
-# TODO: Make accesses actually work
 def commit_to_db(data: dict) -> None:
-    with psycopg2.connect(
-            dbname="benchmarks",
-            user="node-data-sender",
-            password=getenv("DB_PASSWORD"),
-            host="34.90.190.128",
-            port="5432",
-    ) as conn:
-        with conn.cursor() as cursor:
-            insert_query = sql.SQL("""
-                INSERT INTO ft_transfer (
-                    time, git_commit_hash, git_commit_time, num_nodes, node_hardware, 
-                    num_traffic_gen_machines, disjoint_workloads, num_shards, 
-                    num_unique_users, size_state_bytes, tps, total_transactions
-                ) VALUES (
-                    %(time)s, %(git_commit_hash)s, %(git_commit_time)s, %(num_nodes)s, 
-                    %(node_hardware)s, %(num_traffic_gen_machines)s, %(disjoint_workloads)s, 
-                    %(num_shards)s, %(num_unique_users)s, %(size_state_bytes)s, 
-                    %(tps)s, %(total_transactions)s
-                )
-                """)
-            cursor.execute(insert_query, data)
-            conn.commit()
+    chdir(os.path.expanduser("~/nearcore/benchmarks/continous/db/tool"))        
+    json.dump(data, open("tmp.json", "w"))
+    subprocess.run("cargo run -p cli -- insert-ft-transfer tmp.json", shell=True)
+    subprocess.run("rm tmp.json", shell=True)
```

Comment:
You can use a [tempfile](https://docs.python.org/3/library/tempfile.html#examples) for this, something along the lines:

```python
with tempfile.NamedTemporaryFile() as fp:
    json.dump(data, fp)
    fp.close()
    subprocess.run(f"cargo run -p cli -- insert-ft-transfer {fp.name}", shell=True)
```